### DISCUSSION TITLE
Gemini suddenly stopped working

### DISCUSSION THREAD

**Discussion by Delta0001** [end-4/dots-hyprland#966] [General]
I was using gemini fine last month but after I opened up my laptop today, gemini is completely unresponsive. Has anyone experienced this? I've confirmed that my key is correct and active.

**Comment by ghost** [comment]
Me too

**Comment by little-tonii** [comment]
same problem. First, I clear the chat console -> change to balance mode -> ask a question -> problem fixed. I dont know why but it worked for me
reboot maybe needed
remember, only use the flash model, I tried the pro model, it didnt work.
![image](https://github.com/user-attachments/assets/c2eae7cf-d1ae-416a-ae52-758eec444e14)

**Reply by ghost** [reply]
You are right about that.
First you will need to change the model of gemini in ~/.config/ags/services/gemini.js from 1.5 flash to 2.0 flash
Second you will need to check your API at console.cloud.google.com. If there is an announcement that your API is suspended due to violation, that is the main problem

**Reply by end-4** [reply]
i saw this thread and silently changed model to 2.0 flash and default temperature from creative (1) to balanced (0.5)...
i cant say this fixes the problem completely yet without a lot of usage, but i keep wondering why this seems to be a working fix, because on google ai studio the temperature defaults to 1, and the range is [0, 2]...
